src/arch/xtensa/configs/override/jasperlake_chrome.config

```diff
@@ -1,2 +1,2 @@
 CONFIG_CAVS_LPS=y
-CONFIG_CAVS_LPRO=y
+CONFIG_CAVS_USE_LPRO_IN_WAITI=y
```

Comment:
Why are CNL and JSL changed on TGL specific branch? Such changes should be handled by separate commits.

Reply:
I thought even it is TGL specific branch, we should at least make CNL/JSL passed building, no?